account_banking_pain_base/models/account_payment_method.py

```diff
@@ -16,6 +16,7 @@ class AccountPaymentMethod(models.Model):
         "the corresponding unaccented character, so that only ASCII "
         "characters are used in the generated PAIN file.",
     )
+    warn_not_sepa = fields.Boolean(string="Warn If Not SEPA")
```

Comment:
I think the warning shouldn't be selectable. If the payment method is SEPA, then show the warning.

Reply:
I made it an option because I thought that this module may be used in non-sepa countries to produce ISO2022 files. And those users maybe upset to have the warning banner all the time.

Reply:
But the banner should only be shown when the payment method is SEPA. I think you have already checked that on your compute. No problem thus on being "mandatory". It's not going to appear for non SEPA orders.

Reply:
This is not attended.

Reply:
Let me re-explain my point of view in more details. The modules account_banking_sepa_credit_transfer and account_banking_sepa_direct_debit produce ISO20022 files that can be SEPA or not SEPA. We could have called those modules account_banking_iso20022_credit_transfer and account_banking_iso20022_direct_debit, but we chose to call them "sepa"... because, from our point of view of EU users, we use it mostly for SEPA payments. Same for the "code" of the payment method.
But, maybe some users outside of EU use those module to generate non-SEPA ISO20022 files. And they may not want to always have the warning banner. I must say I have no idea if the modules account_banking_sepa_xxx are used by companies outside EU. But I know that ISO20022 is a world-wide standard ; it is not just designed for EU/SEPA.